features/uri-templating.feature

```diff
@@ -20,7 +20,7 @@ Abstract Scenario: samenstellen van een templated url
     Gegeven de OpenAPI specificatie met de volgende servers definitie
```

Comment:
Als gegeven kan je toevoegen dat de adresidentificatie beschikbaar wordt gesteld in de content van de response. Als dat niet zo is heb je niets aan de templated URI.

Reply:
gedaan